gm4_mysterious_midnights/beet.yaml

```diff
@@ -0,0 +1,29 @@
+extend: '../module.yaml'
+
+id: gm4_mysterious_midnights
+name: Mysterious Midnights
+
+data_pack:
+  load: .
+
+pipeline:
+  - data_pack:
+      load: '../lib_forceload'
+
+meta:
+  gm4:
+    description: Full moons gain a element of surprise with Myserious Midnights! This module adds random events during full moons. You might encounter falling stars, or Skeletons wielding poisonous arrows! So you better listen for that howl...
+    required: []
+    recommended:
+      - gm4_midnight_menaces
+    notes: []
+    video: null
```

Comment:
I had no idea that `null` existed in yaml. Why not just omit the field?

Reply:
For the same reason I want to keep fields like `notes` and `recommended` as empty lists. It gives the file a consistent structure and can give hints about what fields might be missing before publishing a module. ~~Also I just came up with this explanation as I'm typing this~~